app/helpers/application_helper/button/tenant_edit.rb

```diff
@@ -1,7 +1,5 @@
 class ApplicationHelper::Button::TenantEdit < ApplicationHelper::Button::Basic
-  needs :@record
-
   def disabled?
-    @record.source
+    @record.try!(:source)
```

Comment:
There is no need to modify this class, because it is not needed to add this to summary toolbar.